rpm/scratchbox2.spec

```diff
@@ -58,3 +60,4 @@ install -D -m 644 utils/sb2.bash %{buildroot}/etc/bash_completion.d/sb2.bash
 %ifarch x86_64
 /usr/lib32/libsb2/*
 %endif
+%{_exec_prefix}/lib64
```

Comment:
libsb2 is also installed under x86 build targets, so this may not be a good place for packaging the symlink?

Reply:
I didn't notice that :( Any suggestions for a better place?

Reply:
I fail to see an ideal place, but maybe sdk-tooling-configs would be a not so bad place.

Reply:
Should the symlink maybe be in the tooling since this specific to this to our targets right now but nowhere else.
Also this won't/doesn't make sense under x86_64.